task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "appstudio, hacbs"
+  name: fbc-validation
+spec:
+  params:
+    - name: IMAGE_URL
+      description: the fully qualified image name
+  results:
+    - name: HACBS_TEST_OUTPUT
+      type: string
+  steps:
+    - name: check-labels-and-binaries
+      image: quay.io/redhat-appstudio/hacbs-test:latest
+      workingDir: /work
+      securityContext:
+        runAsUser: 1000
+      env:
+        - name: HOME
+          value: /work
+      resources:
+        limits:
+          memory: 4Gi
+          cpu: 2
+        requests:
+          memory: 512Mi
+          cpu: 10m
+      script: |
+        ### Check if label exists > If yes try to extract binaries with configs > check binaries functionality > check opm validate ###
+        conffolder=$(skopeo inspect --format '{{ 1 | or (index .Labels "operators.operatorframework.io.index.configs.v1") }}' docker://"$(params.IMAGE_URL)")
```

Comment:
In my task [HACBS-1436](https://issues.redhat.com/browse/HACBS-1436), I would patch a step as below in pipelines/fbc-builder/patch.yaml to check this label after https://github.com/redhat-appstudio/build-definitions/pull/249 is merged
```
- name: fbc-label-check
  when:
  - input: $(params.hacbs)
    operator: in
    values: ["true"]
  runAfter:
    - sanity-inspect-image
  taskRef:
    name: sanity-label-check
    version: "0.1"
  params:
  - name: POLICY_NAMESPACE
    value: fbc_checks
  workspaces:
  - name: workspace
    workspace: workspace
```
So I think you don't need to check the label again and just run this validation after [HACBS-1436](https://issues.redhat.com/browse/HACBS-1436) check the label.

Reply:
Same as above, when #249 gets merged, I will add those also.

Reply:
included